src/app/components/navbar/navbar.html

```diff
@@ -17,61 +17,46 @@
         ui-sref="home"
       >
         <span class="fa fa-home"></span>
-        <span ng-bind="::navbarCtrl.name"></span>
+        <span ng-bind="::name"></span>
       </a>
     </div>
 
     <div
-      ng-if="navbarCtrl.navbarState.items.length"
+      ng-if="navbarState.items.length"
       class="collapse navbar-collapse"
-      collapse="navbarCtrl.navbarState.collapsed"
-    >
+      collapse="navbarState.collapsed"
+      >
+
       <ul class="nav navbar-nav">
         <li
-          ng-repeat="item in navbarCtrl.navbarState.items track by item.name"
+          ng-repeat="item in navbarState.items track by item.name"
           ui-sref-active="active"
           eha-show-for-role="item.roles"
-        >
+          >
           <a
-            ng-click="navbarCtrl.toggleCollapse()"
+            ng-click="toggleCollapse()"
             ui-sref="{{item.name}}"
             ng-bind="::item.label"
-          ></a>
+            ></a>
         </li>
       </ul>
-      <ul class="nav navbar-nav navbar-right dropdown">
-        <div class="btn-group" dropdown is-open="navbarCtrl.isopen">
-          <button type="button" class="btn btn-default btn-lg" dropdown-toggle ng-disabled="disabled">
-            <i class="fa fa-user fa-lg"></i>
-            <span ng-bind="navbarCtrl.navbarState.username"></span>
-            <span class="caret"></span>
-          </button>
-          <ul class="dropdown-menu" role="menu">
-            <li>
-              <a class="pointer" ng-click="navbarCtrl.toggleCollapse(); navbarCtrl.logout()">Logout</a>
-            </li>
-          </ul>
-        </div>
-      </ul>
+    <eha-dropdown-menu
```

Comment:
Yes, it looks like that, but its all about code reuse. i also used it in the username/logout menu, and i  think we might still have another menu, who knows